fdbclient/ManagementAPI.actor.cpp

```diff
@@ -52,6 +52,11 @@ std::map<std::string, std::string> configForToken( std::string const& mode ) {
 		return out;
 	}
 
+	if (mode == "locked") {
+		out[databaseLockedKey.toString()] = deterministicRandom()->randomUniqueID().toString();
```

Comment:
This is a different format than what we usually use. I see that the provisional master doesn't support atomic operations. I know that the provisional master doesn't support atomics, but I think this value should at least have the correct length (we might even know what the version of the first recovery transaction is? In that case we could even put the correct version there - though imho setting it to version `0` to indicate that the database was configured locked would be simple and useful enough)

Reply:
I think the first commit actually does support atomic ops. I am using the correct format for the databaseLockedKey. The actual transaction interprets this config key being set as an instruction to do the normal atomic op. I'll add a comment explaining.